dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -22,11 +22,26 @@
       Notes for yourself or other Levelbuilders. Viewable here and here alone.
     = f.text_area :notes, rows: 6, class: "input-block-level"
 
-.field
-  = f.label 'bubble_choice_description'
-  %p
-    Description of this level and why a student would want to choose it as part of a BubbleChoice progression.
-  = f.text_area :bubble_choice_description, rows: 3, class: "input-block-level"
+
+- if !@level.is_a?(BubbleChoice)
```

Comment:
I think this would be a good time to extract the contents of this `if` block into a new `levels/editors/_bubble_choice.html.haml` file which you include from `levels/edit.html.haml`